src/main/resources/ui.properties

```diff
@@ -93,15 +93,15 @@ feedbackSend = send
 title = Qabel Desktop Client
 browse = File browse
 faqUrl=https://qabel.de/de/faq
-aboutHeadline=Legal
+aboutHeadline=legal Stuff:
```

Comment:
I know it's in the mockup (https://cloud.githubusercontent.com/assets/3028703/15392861/1ccdbf40-1dca-11e6-9cbb-8e40ecd48339.png) but I think @thechauffeur should ack that we are changing to a more informal language